src/command/cmd_defs.c

```diff
@@ -2729,6 +2729,18 @@ static const struct cmd_t command_defs[] = {
               "/privacy logging off",
               "/privacy os off")
     },
+
+    { CMD_PREAMBLE("/redraw",
+                   parse_args, 0, 0, NULL)
+      CMD_MAINFUNC(cmd_redraw)
+      CMD_TAGS(
+              CMD_TAG_UI)
+      CMD_SYN(
+              "/redraw")
+      CMD_DESC(
+              "Redraw User Interface. Might be used in case if some other program interrupted and interface is \"broken\"." )
```

Comment:
I'm not sure `in case if` is correct.  I think the `if` can be canceled. Personally I probably would just use the first sentence.

Reply:
I also believe `User` and `Interface` should be lower case.

Reply:
```suggestion
              "Redraw user interface. Can be used when some other program interrupted profanity or wrote to the same terminal and the interface looks \"broken\"." )
```

I also would just use the first sentence, but that's because I'm lazy :)

Here we're writing the help text for our users, which also shows up in the man pages, and I like the idea to include the "why" some command exists.